app/react/Pages/components/Script.js

```diff
@@ -13,6 +13,7 @@ class Script extends Component {
 
   componentDidMount() {
     this.appendScript();
+    window.onerror = this.props.onError;
```

Comment:
@mfacar this seems like will have a similar problem the script had before.
window.onerror will not be changed until you navigate to a page, from that point window.onerror will always be this.props.onError, even if you navigate out of the page, should we remove callback using the same logic as the script itself ?
see ```componentDidUpdate``` and ```componentWillUnmount``` or maybe there is another way to catch errors on the script ?

Reply:
Yes, thanks, I changed to `window.removeEventListener` instead and used the same logic than script